app/actions/websocket/group.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {fetchGroup, fetchGroupsForChannel, fetchGroupsForMember, fetchGroupsForTeam} from '@actions/remote/groups';
+import DatabaseManager from '@database/manager';
+import {logError} from '@utils/log';
+
+type WebsocketGroupMessage = WebSocketMessage<{
+    group?: string; // type Group
+}>
+
+const handleError = (serverUrl: string, e: unknown, msg: WebsocketGroupMessage) => {
```

Comment:
Seems like a nice fallback, but when do you expect this to be executed? From the code I see that whenever `msg.data.group` is not well formed. Are we expecting that to ever happen?

Reply:
I hope not, but I think we try to make sure we cover any eventualities from the server side.